src/constants.h

```diff
@@ -45,8 +45,10 @@ constexpr uint32_t SERVER_UNRESPONSIVE_COOLDOWN_SEC = 300;
 constexpr uint32_t BENCHMARK_MAX_BITS_UPLOAD = 80000000;  // 10 Megabyte
 constexpr uint32_t BENCHMARK_MAX_DURATION_PING = 3000;
 constexpr uint32_t BENCHMARK_MAX_DURATION_TRANSFER = 15000;
-constexpr uint32_t BENCHMARK_THRESHOLD_SPEED_FAST = 25000000;    // 25 Megabit
-constexpr uint32_t BENCHMARK_THRESHOLD_SPEED_MEDIUM = 10000000;  // 10 Megabit
+constexpr uint32_t BENCHMARK_THRESHOLD_DOWN_FAST = 25000000;    // 25 Megabit
+constexpr uint32_t BENCHMARK_THRESHOLD_DOWN_MEDIUM = 10000000;  // 10 Megabit
+constexpr uint32_t BENCHMARK_THRESHOLD_UP_FAST = 5000000;       // 5 Megabit
+constexpr uint32_t BENCHMARK_THRESHOLD_UP_MEDIUM = 2500000;     // 2.5 Megabit
```

Comment:
Very German definition of what is fast internet is :D - but I think 25mbit are sufficient for most people?

Reply:
I agree. I’d be generally cautious to call a 25 Mbit/s fast, but since we use that threshold mainly in order to communicate what the current connection is capable of that should cover normal use cases?